app/src/components/fields/fieldErrorHint/fieldErrorHint.scss

```diff
@@ -23,6 +23,10 @@
   &.type-top {
     bottom: 100%;
   }
+
+  & .type-top {
```

Comment:
There's no point in using nested selector because there's already a special class for target element (`.hint-content`). Please move this style (without selector) below the line 48.

Reply:
But wouldn't that affect they bottom type ones as well? (Same issue as the first iteration of this PR)

Reply:
fixed